### ISSUE TITLE
Can't install

### ISSUE THREAD

**Issue by paul-frost** [jaseclamp/madison_alttags#1]
Problems with the module name & description causing the modules page to break when you try and install it.
Looking at the code for where that might be fixable I noticed several mentions of Detour Pro in the upd file. I'm not a programmer, but it looks to me like it might drop any detour related tables on uninstalling.

I'm glad it didn't install now, as I may have lost my detour pro data.

**Comment by jaseclamp** [comment]
Indeed, major faux pas on my part paul. Thanks for checking, raising this and saving others from experiencing that.  I based the view on detour because it was similar to what I wanted to accomplish and I'm not a programmer per say. I've updated the code to ensure it doesn't mention, interact with or change anything on detour (https://github.com/jaseclamp/madison_alttags/commit/d55f2a1d04b98523714e86f27a3a1e627dd4dbe1). I created the extension because I couldn't find anything like it out there and thought there should be something like it due to it's importance with SEO. Its been working well for us in terms of it being fast to go through and update alt tags, etc. Thanks again ~J
